### PR TITLE
Update supportconfig file extension to txz

### PR DESCRIPTION
**Pull request by gosipyan**

### CODE REVIEW

**Comment by jsuchome** [comment]
What's the reason for this? Can we have a reference why the requirement has changed?

**Comment by gosipyan** [comment]
@jsuchome  Currently on all CI deployed cloud setups C7/8/9  I see supportcofings with  ".txz" extensions  and from changelog looks like there was change related to compression method recently https://build.opensuse.org/package/view_file/SUSE:SLE-12:Update/supportutils/supportutils.changes?expand=1

**Comment by jsuchome** [comment]
@gosipyan Thanks! Can you add the changelog explanation to the commit message?

File: scripts/jenkins/cloud/ansible/roles/supportconfig/defaults/main.yml [hunk 1]
```diff
@@ -21,4 +21,4 @@ supportconfig_pkgs:
   - supportutils-plugin-suse-openstack-cloud
 
 supportconfig_filename: "{{ cloud_env }}-{{ (inventory_hostname in groups['deployer_virt']) | ternary('admin', inventory_hostname) }}"
-supportconfig_filepath: "/var/log/nts_{{ supportconfig_filename }}.tbz"
```
Comment (stefannica) [inline scripts/jenkins/cloud/ansible/roles/supportconfig/defaults/main.yml]: If we don't use .tbz, then we do not need to install the bzip2 package above anymore

**Comment by stefannica** [comment]
Can you also take care of mkcloud jobs in the same PR ? (see https://github.com/SUSE-Cloud/automation/blob/master/scripts/mkcloud#L234-L252).

**Comment by gosipyan** [comment]
Testing 
https://ci.suse.de/job/cloud-crowbar8-job-gate-ha-deploy-x86_64/148/

**Pull request merged**